application/src/main/kotlin/application/StubCommand.kt

```diff
@@ -116,7 +112,8 @@ class StubCommand : Callable<Unit> {
             contractSources = when (contractPaths.isEmpty()) {
                 true -> {
                     logger.debug("No contractPaths specified. Using configuration file named $configFileName")
-                    specmaticConfigPath = File(Configuration.globalConfigFileName).canonicalPath
+                    specmaticConfigPath = getConfigFileName()
+                    Configuration.globalConfigFileName = getConfigFileName()
```

Comment:
Let's move line 116 after line 114. So that, we don't have to make the change ` specmaticConfigPath = getConfigFileName()`

Test it once.